pages/common/direnv.md

```diff
@@ -0,0 +1,24 @@
+# direnv
+
+> Direnv is an extension for your shell. It augments existing shells with a new feature that can load and unload environment variables depending on the current directory.
+> More information: <https://github.com/direnv/direnv>.
+
+- Grant direnv to load the given .envrc:
```

Comment:
I'm confused by this. What does this example actually do?

Reply:
what example? `direnv allow`?

Reply:
`direnv allow` by default `direnv` does not load any env files. we have to manually give this command for the first time to direnv to pick it up, because of security thing.

Reply:
Ah, I see. Thanks for the explanation :-)